rpc/src/module/indexer.rs

```diff
@@ -63,6 +63,7 @@ pub trait IndexerRpc {
     ///
     /// ## Returns
     ///
+    /// > If the number of objects is less than the requested `limit`, it indicates that these are the last page of get_cells.
```

Comment:
Why quote (`> `) here? If the text messes up with the following list, try to separate them with a blank line.

```suggestion
    /// If the number of objects is less than the requested `limit`, it indicates that these are the last page of get_cells.
    ///
```